service/src/integrationTest/kotlin/fi/espoo/evaka/messaging/BulletinIntegrationTest.kt

```diff
@@ -348,6 +348,21 @@ class BulletinIntegrationTest : FullApplicationTest() {
         assertEquals(2, childWithTwoReceiverPersons.receiverPersons.size)
     }
 
+    @Test
+    fun `Sender options endpoint works`() {
+        val (_, res, result) = http.get("/bulletins/sender-options?unitId=$unitId")
+            .asUser(supervisor)
+            .responseObject<List<String>>()
+
+        assertEquals(200, res.statusCode)
+
+        val senderOptions = result.get()
+
+        assertEquals(3, senderOptions.size)
+
+        assertEquals(setOf("Elina Esimies", "Testaajat", "Test Daycare"), senderOptions.toSet())
+    }
```

Comment:
I'd also like to see the test repeated for the second unit (that it gets the correct unit and still the same user) -- just as a regression/sanity test.